### PR TITLE
Subdivide the navbar and add info about v1.0RC

### PR DESCRIPTION
**Pull request by bensigelman**
The revised sidebar:

image

### CODE REVIEW

**Comment by bensigelman** [comment]
(I want to have the v1.0RC copy up there for the note I'm about to send to distributed-tracing@)

I assume this is uncontroversial and will probably merge myself in an hour or two... happy to revisit if there are concerns!

File: _includes/sidebar.html [hunk 1]
```diff
@@ -13,7 +13,18 @@
         `layout: page` in the front-matter. See readme for usage.
       {% endcomment %}
 
-      {% for page in site.pages_list %}
+      <div class="sidebar-group">The basics</div>
```
Comment (yurishkuro) [inline _includes/sidebar.html]: I'd drop this title, not needed.

File: apis.md [hunk 2]
```diff
@@ -15,3 +15,11 @@ OpenTracing APIs are **in progress** for the following platforms:
 Please refer to the README files in the respective per-platform repositories for usage examples.
 
 PHP, iOS, and Ruby are next on our list, though community contributions are welcome for other languages at any time.
+
+<div id="v1rc"></div>
+
+### What does "1.0RC" mean?
+
+It means that the contributors have spent a while thinking through the APIs from above and below, prototyped bindings to several tracing systems and many calling contexts, and have settled on the scope and exposed by the various OpenTracing APIs.
```
Comment (yurishkuro) [inline apis.md]: "and exposed" - not sure if

**Comment by yurishkuro** [comment]
lgtm, less minor comments.

It's unfortunately that Github link is under For Implementors, technically it applies to all.

**Comment by bensigelman** [comment]
Easy to move the github link up to "all"...

I'll make sure it looks reasonable without the "The basics" header and merge.

File: apis.md (hunk 2 above)
Comment (bensigelman) [inline apis.md]: reworded this more generally...

**Comment by yurishkuro** [comment]
@bensigelman oh-oh, I get this on my Mac, and the sidebar does not scroll up.

image

**Comment by yurishkuro** [comment]
maybe we can move the tag line from under the logo onto the top of the home page? I can send PR

image

**Pull request merged**